### PR TITLE
V1.3.4 adaptive sample rates

### PR DESCRIPTION
**Pull request by brentpicasso**
* Sample rates for various channel types now reflect capabilities of device (MK1 vs MK2)'
* PEP8 compliance improvements

### CODE REVIEW

File: autosportlabs/racecapture/views/configuration/baseconfigview.py [hunk 1]
```diff
@@ -57,17 +57,18 @@ def on_tracks_updated(self, track_manager):
         
 
 class LazyloadAccordionItem(AccordionItem):
-    def __init__(self, **kwargs):
+    def __init__(self, max_sample_rate, builder, channel_index, **kwargs):
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/configuration/baseconfigview.py]: :+1: So much easier to understand this way :)

**Comment by brentpicasso** [comment]
Note, RCP MK2 rejects sample rates above 200Hz. I believe this is a firmware problem.

File: autosportlabs/racecapture/views/configuration/baseconfigview.py (hunk 1 above)
Comment (brentpicasso) [inline autosportlabs/racecapture/views/configuration/baseconfigview.py]: \o/

**Comment by ryandoherty** [comment]
Is there any easy way to look at a diff that doesn't include PEP8/whitespace changes?

**Comment by brentpicasso** [comment]
eh..., look at the individual commits to see what functional changes I made. I kept many of functional changes separate.

**Comment by ryandoherty** [comment]
Seems fine to me, 80% of this is PEP8 or just passing max_sample_rate around.

Unfortunately I can't just look at individual commits and comment on them, because things I comment on in one commit may have changed in another. It would help if in the future PRs for big PEP8 or whitespace changes are individual PRs so it's easier to review. 

Anyway, glad this is in here!

**Comment by brentpicasso** [comment]
Thanks. Yeah, this was a broad stroke but not too disruptive. Lots of PEP8 improvements that seemed to be in the way.

**Pull request merged**